includes/digilan-activator.php

```diff
@@ -148,20 +148,24 @@ public static function get_ap_settings()
         if (empty($settings->get('access-points')[$hostname])) {
             _default_wp_die_handler('No such hostname.');
         }
-        $intervals = $settings->get('access-points')[$hostname]['schedule'];
+        $ap_setting_model = $settings->get('access-points')[$hostname];
+        $ap_setting_array = $ap_setting_model->get_config();
+        $ap_settings = $ap_setting_array['ap_settings'];
+        $global_settings = $ap_setting_array['global_settings'];
```

Comment:
global c est partager entre les ap ?

donc shared_settings ? 

et ap c estpour un ap specific ?  mais ou est le if ( not ap_ ) use global ?